package.json

```diff
@@ -30,14 +39,14 @@
     },
     "scripts": {
         "docker:up": "cd packages/backend && docker-compose up -d",
-        "build": "cd packages/lib/ && yarn build && cd ../react && yarn build && cd ../web && yarn build && cd ../backend && yarn build && cd ../offchain-resolver && yarn build",
-        "start": "cd packages/backend && yarn start",
+        "build": "yarn workspaces run build",
+        "start": "cd ./packages/backend && yarn start",
         "resolver": "cd packages/offchain-resolver && yarn start",
         "lint": "eslint '*/**/*.{js,ts,tsx}' --fix",
         "format": "prettier --write '*/**/*.{js,ts,tsx,json}'",
         "format:check": "prettier --check '*/**/*.{js,ts,tsx,json}'",
         "prepare": "husky install",
-        "test": "npm run docker:up && ./test-all.sh"
+        "test": "./test-all.sh"
```

Comment:
This should be replaced with something like `yarn workspaces run test`
test-all.sh can then be deleted.